modules/olmv1-adding-a-catalog.adoc

```diff
@@ -25,8 +34,10 @@ spec:
     type: image
     image:
       ref: registry.redhat.io/redhat/redhat-operator-index:v{product-version} <1>
+      pullSecret: <pull_secret_name> <2>
 ----
 <1> Specify the catalog's image in the `spec.source.image` field.
+<2> Catalogs hosted on a secure registry, such as `registry.redhat.io`, require a pull secret scoped to the `openshift-catalog` namespace.
```

Comment:
Maybe:
```suggestion
<2> Optional: Catalogs hosted on a secure registry, such as `registry.redhat.io`, require a pull secret scoped to the `openshift-catalog` namespace.
```

Reply:
I tweaked the sentence a bit, but I went a different direction. I think that this is a conditional requirement as opposed to optional. So I rephrased the sentence to clarify that.